rdr_service/model/study_nph.py

```diff
@@ -414,3 +414,24 @@ class ParticipantOpsDataElement(NphBase):
 
 event.listen(ParticipantOpsDataElement, "before_insert", model_insert_listener)
 event.listen(ParticipantOpsDataElement, "before_update", model_update_listener)
+
+
+class DlwDosage(NphBase):
+    __tablename__ = "dlw_dosage"
+
+    id = Column("id", BigInteger, autoincrement=True, primary_key=True)
+    created = Column(UTCDateTime)
+    modified = Column(UTCDateTime)
+    ignore_flag = Column(TINYINT, default=0)
+    participant_id = Column(BigInteger, ForeignKey("participant.id"), nullable=False)
+    module = Column(Enum(ModuleTypes), nullable=False)
+    visit_period = Column(Enum(VisitPeriod), nullable=False)
+    batch_id = Column(String(32), nullable=False)
+    participant_weight = Column(String(32), nullable=False)
+    dose = Column(String(32), nullable=False)
+    calculated_dose = Column(String(32), nullable=False)
```

Comment:
These fields store values like `{... "participantweight": "56.38", "dose": "84", "calculateddose": "84.57",..}`. is it ok to store them as string?

Reply:
@shahwaiz14 I think casting them as strings is fine, its a pattern that we often do.

Reply:
Got it, thanks Darryl